packages/web-components/src/drawer-body/drawer-body.template.ts

```diff
@@ -0,0 +1,25 @@
+import { ElementViewTemplate, html, ref } from '@microsoft/fast-element';
+import type { DrawerBody } from './drawer-body.js';
+
+/**
+ * The template for the Drawer component.
+ * @public
+ */
+export function drawerBodyTemplate<T extends DrawerBody>(): ElementViewTemplate<T> {
+  return html<T>`
+    <div class="drawer-body" part="drawer-body">
```

Comment:
Same as above, not sure we need the `.drawer-body` class, that can just be the parent `<fluent-drawer-body>` element.